app/domain/logs.rb

```diff
@@ -262,6 +262,35 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnJwt
+
+      FetchingIssuerConfigurationValue = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching issuer authenticator configuration value",
```

Comment:
This is progressive....
End with ellipsis (...)

"Fetching "issuer" value from authenticator configuration..."

Reply:
(I don;t know what it means but the english is fine . Is it the **authenticator** configuration value of the issuer?

Reply:
Changed to 
Fetching "issuer" value from authenticator configuration...

Reply:
It means fetching from policy